### PR TITLE
Add hex colors

### PR DESCRIPTION
**Pull request by justynjozwiak**

### CODE REVIEW

**Comment by rstacruz** [comment]
Thanks for the PR! Can you add a simple test?

File: lib/ffaker/color.rb [hunk 1]
```diff
@@ -9,5 +9,8 @@ def name
       NAMES_LIST.sample
     end
 
+    def hex
+      "##{(0..5).map { (('a'..'f').to_a + (0..9).to_a).sample }.join}"
+    end
```
Comment (rstacruz) [inline lib/ffaker/color.rb]: maybe cache the hex digits like so:

```rb
HEX_DIGITS = (('a'..'f').to_a + (0..9).to_a)

def hex
  "##{6.times.map { HEX_DIGITS.sample }.join}"
end
```

Comment (justynjozwiak) [inline lib/ffaker/color.rb]: @rstacruz thanks for the tip, fixed

**Comment by justynjozwiak** [comment]
added simple test

File: test/test_color.rb [hunk 2]
```diff
@@ -6,4 +6,8 @@ class TestColor < Test::Unit::TestCase
   def test_name
     assert_match /\A[a-z]+\z/, FFaker::Color.name
   end
+
+  def test_hex
+    assert_match((/(?<=#)(?<!^)\h{6}/), FFaker::Color.hex)
```
Comment (marocchino) [inline test/test_color.rb]:
```ruby
assert_match(/\A#[\da-f]{6}\z/, FFaker::Color.hex)
```

**Comment by marocchino** [comment]
duplication of #260

**Pull request closed without merging**